src/index.jsx

```diff
@@ -5,7 +5,6 @@ import ReactDOM from 'react-dom/client';
 // Component Imports
 import App from './App';
 // Other Imports
```

Comment:
We technically could remove this comment as well since it's comment was originally used for the styling import.